test/libsolidity/semanticTests/arithmetics/checked_add_v2.sol

```diff
@@ -0,0 +1,14 @@
+pragma abicoder v2;
+contract C {
+    // Input is still not checked - this needs ABIEncoderV2!
```

Comment:
Is this comment a left-over from something older? This test does have the v2 pragma.

Reply:
It is a copy of the `checked_add_v1.sol` file.

Reply:
Fixed.